etc/docker/elyra/start-elyra.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Needs license header

Reply:
haven't we fixed this on lint @ajbozarth ?

Reply:
the license header linting is only for the front end (.ts,.tsx,.js files), @kevin-bates wrote the back end lint and I'm unsure if he added that, it could also be because this script is not in the elyra dir and the backend lint only runs there (front end lint runs on the whole repo)

Reply:
Yes - currently flake8 runs on the `elyra` tree and only against `.py` files.

Reply:
Please add a header to https://github.com/elyra-ai/elyra/blob/master/etc/scripts/protect-metadata.sh while you're at it. :smile: